pages/1.13/security/oss/authentication/out-of-band-verification/index.md

```diff
@@ -0,0 +1,96 @@
+This example uses the Python [PyJWT module](https://pyjwt.readthedocs.io/en/latest/), auth token verification, and extraction of the user ID:
+
+```python
+# `token` is the string holding the authentication token.
+# `public_key` is an RSA public key object of the cryptography module.
+payload = jwt.decode(token, public_key, algorithm='RS256')
+uid = payload['uid']
+```
+
+The decode method verifies the token signature and expiration time and raises an exception if the token is invalid.
+
+## Complete token verification example
+This example validates an authentication token. Here is the example token
+
+```json
+eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJleHAiOjE0Njc5ODU0NjYsInVpZCI6InBldGVyIn0.lsLJx2WsX99HF96CizMOcZpMIgbjGDBHvFZCGeNDsM-xZQzHQJHo_UA8WodQ52o8uBJ2CY983DhJdIH2Gfc_fbZtYGvUx-IvQnHFbUBd8qBN0A_4BQHeNINFUKdVQuJsbsW-uVj-w0q3RAFwO5DPPc2ppwIjkeQbgGP1ZN-2-uV6Jow04cdkq4jcODsD1y0v4EmIBPLQil0HU2B95IHtlBNN7haTUkCksXE-43BHy4ErboySeq6VgkwLpw_Pi8n236kZ2-GobSmhA-BpjbkO3uGLHrYUfJjrJyiPM2_PZQMHY80-m5sMMMQ9m1Ciag2Cw74JKGfJ3qMW3j3z2Hm7GQ
+```
+
+Here is Python code for performing the validation, following the instructions given above:
+
+```python
+>>> import jwt
+>>> import requests
+>>> from cryptography.hazmat.backends import default_backend
+>>> from cryptography.hazmat.primitives.asymmetric import rsa
+>>> from jwt.utils import base64url_decode, bytes_to_number
+
+>>> keys = requests.get('http://localhost:8101/acs/api/v1/auth/jwks').json()['keys'][0]
+>>> keys
+{'kty': 'RSA', 'n': 'sybUYxu3TXxXAgG_Eq72tKxE7xhGFgL14g5OGryDtE5dBL8frAoSsI4D7tSKR2pLbOlT68YJbYLUHxoju0E_NB9htjKEsay4t3WXoXQ-XsDM4Zz22H6HfDG6CCcvGb2DoQP0R2je1HJDA56_BoR8shZMxHbrX1WgQURtGygMD7bQY95qmHZYRPlq13-pR5Jnu70OMmFlbl-_o-ag1JfndTJPtx75IalCgy_h_itHLDPhdTfypAJeiewCOUZd9nNa1j19M-xeqlZonlRABqiH0e-vQVWCeW5FZ0HJamIjd2VifhRCp0fSAgCdCQdrY6HdI3h6egpn6z4gwkwXBfczww', 'kid': '55fb61042768f62ea3b06778c6043f7c8c92769a0c248076a2995dfd50c4acb9', 'use': 'sig', 'alg': 'RS256', 'e': 'AQAB'}
+
+>>> exponent_bytes = base64url_decode(keys['e'].encode('ascii'))
+>>> exponent_int = bytes_to_number(exponent_bytes)
+
+>>> modulus_bytes = base64url_decode(keys['n'].encode('ascii'))
+>>> modulus_int = bytes_to_number(modulus_bytes)
+
+>>> public_numbers = rsa.RSAPublicNumbers(n=modulus_int, e=exponent_int)
+>>> public_key = public_numbers.public_key(backend=default_backend())
+
+>>> authtoken = "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJleHAiOjE0Njc5ODU0NjYsInVpZCI6InBldGVyIn0.lsLJx2WsX99HF96CizMOcZpMIgbjGDBHvFZCGeNDsM-xZQzHQJHo_UA8WodQ52o8uBJ2CY983DhJdIH2Gfc_fbZtYGvUx-IvQnHFbUBd8qBN0A_4BQHeNINFUKdVQuJsbsW-uVj-w0q3RAFwO5DPPc2ppwIjkeQbgGP1ZN-2-uV6Jow04cdkq4jcODsD1y0v4EmIBPLQil0HU2B95IHtlBNN7haTUkCksXE-43BHy4ErboySeq6VgkwLpw_Pi8n236kZ2-GobSmhA-BpjbkO3uGLHrYUfJjrJyiPM2_PZQMHY80-m5sMMMQ9m1Ciag2Cw74JKGfJ3qMW3j3z2Hm7GQ"
+>>> payload = jwt.decode(authtoken, public_key, algorithm='RS256')
+>>> payload
+{'uid': 'peter', 'exp': 1467985466}
+```
+
+The response indicates that this is a valid authentication token for `peter`.
```

Comment:
Rewrite #96 (If response and output refer the same).
The output indicates a valid authentication for `peter`